src/modules/events/actions/log-handlers.js

```diff
@@ -51,6 +51,14 @@ export const handleTokensTransferredLog = log => (dispatch, getState) => {
   }
 }
 
+export const handleTokensMintedLog = log => (dispatch, getState) => {
+  dispatch(updateAssets())
+}
+
+export const handleTokensBurnedLog = log => (dispatch, getState) => {
+  dispatch(updateAssets())
+}
```

Comment:
if (getState().loginAccount.address === log.target || getState().loginAccount.address === log.sender) {
    dispatch(updateAssets())
  }

What about making sure the user is interested in the change before updating assets?